yarn-project/end-to-end/src/e2e_blacklist_token_contract/minting.test.ts

```diff
@@ -85,21 +85,18 @@ describe('e2e_blacklist_token_contract mint', () => {
 
     describe('Mint flow', () => {
       it('mint_private as minter and redeem as recipient', async () => {
+        const balanceBefore = await asset.methods.balance_of_private(adminAddress).simulate({ from: adminAddress });
```

Comment:
It was not justified to use the low level getNotes here so I replace with the balance_of_private call instead.